package.json

```diff
@@ -8,7 +8,8 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook --output-dir docs",
     "deploy-storybook": "storybook-to-ghpages",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
```

Comment:
nit: kebab-case

Reply:
This is a named key, not something I named myself.